### ISSUE TITLE
Lodestar: Social Profiles Block Displaying as Bulleted List on Front Page Panel

### ISSUE THREAD

**Issue by mdrockwell** [Automattic/themes#1991]
## Steps to replicate

1. Activate Lodestar
2. Create a new page
3. Add the Social Profile block into the contents of the page and add some profile links
4. Navigate to My Site > Design > Customize > Theme Options
5. Select that page from one of the Panels (Panel 1, Panel 2, etc.)

## Result

The Social Profile block appears in a bulleted list using standard buttons.

![Screen Shot 2020-05-07 at 17 59 14](https://user-images.githubusercontent.com/42454301/81348673-80444380-908c-11ea-8979-dc6be0c1b6ed.png)

## Expected

The Social Profiles should appear as they do in other pages throughout the site.

<img width="759" alt="Screen Shot 2020-05-07 at 18 00 00" src="https://user-images.githubusercontent.com/42454301/81348717-9ce07b80-908c-11ea-8fa5-a6d8b79ecc1c.png">

* User report: #20916908-hc
* Follow-up ticket: #2954409-zd

**Comment by nielslange** [comment]
Hello @mdrockwell 👋 _"It's a feature, not a bug!"_ Joke aside, I've investigated this issue and found out what caused it. The _Social Profiles Block_ comes from the CoBlocks plugin. The reported issue appears, as the styles of the CoBlocks plugin are not loaded on the homepage. Whike this sounds like a bug, it is actually a feature of the CoBlocks plugin to ensure that the frontend styles are only loaded when the corresponding page contains at lest one block of the CoBlocks plugin. This functionality had been defined in https://github.com/godaddy-wordpress/coblocks/blob/25b621074f208448848b2c2c8159ae2ad4baa60c/includes/class-coblocks-block-assets.php#L79-L80

When using the front page panels of the Lodestar theme, the CoBlocks plugin does not check the embedded pages or posts but the static homepage instead. If the static homepage contains a CoBlocks block, then the frontend styles are loaded and the social profile blocks are formated as expected. However, if the static homepage does not contain a CoBlocks block, the frontend styles will not be loaded. 

**Possible solutions:**

1. Add at least one CoBlocks block on the static homepage to ensure that the frontend styles are loaded
2. Replace the Social Profile Block with the [Social Links Block](https://wordpress.com/support/wordpress-editor/blocks/social-links-block/)
3. Report this issue on https://github.com/godaddy-wordpress/coblocks/issues and hope that it gets addressed

As for the 3rd option, it might be rather tricky to detect if one of the embedded pages/posts contains a CoBlocks block and removing the check, that I pointed out above, would mean that the styles are loaded even when no CoBlocks blocks are available on a page/post, which will impact the performance of the site.

I'm going to close this issue for now, as it's not caused by the Lodestar theme, but let me know if you have any questions regarding my explanation above. 😉
